composite-actions/nodejs-generic-api/test-opa/action.yaml

```diff
@@ -50,4 +53,4 @@ runs:
           *Build failed for ${{ github.repository }}: ${{ github.workflow }}* :heavy_exclamation_mark:
```

Comment:
is there any reason to have this in test-opa action? will be a lot of nise from oull requests i think. i would leave this for staging and prod. I would probably expect different message for prod vs staging